### PR TITLE
rework work time tables

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

File: src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210805151400_DropWorkTimeTableAddWorkTables.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                columns: table => new
+                {
+                    Id = table.Column<Guid>(nullable: false),
+                    UserId = table.Column<Guid>(nullable: false),
+                    ProjectId = table.Column<Guid>(nullable: false),
+                    WorkTimeMonthLimitId = table.Column<Guid>(nullable: false),
+                    Year = table.Column<int>(nullable: false),
+                    Month = table.Column<int>(nullable: false),
+                    UserHours = table.Column<float>(nullable: false),
+                    ManagerHours = table.Column<float>(nullable: false),
```
Comment (BelehovEgor) [inline src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210805151400_DropWorkTimeTableAddWorkTables.cs:28]: nullable true

File: src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210805151400_DropWorkTimeTableAddWorkTables.cs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                name: DbWorkTime.TableName,
+                columns: table => new
+                {
+                    Id = table.Column<Guid>(nullable: false),
+                    UserId = table.Column<Guid>(nullable: false),
+                    ProjectId = table.Column<Guid>(nullable: false),
+                    WorkTimeMonthLimitId = table.Column<Guid>(nullable: false),
+                    Year = table.Column<int>(nullable: false),
+                    Month = table.Column<int>(nullable: false),
+                    UserHours = table.Column<float>(nullable: false),
```
Comment (BelehovEgor) [inline src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210805151400_DropWorkTimeTableAddWorkTables.cs:27]: true

File: src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210805151400_DropWorkTimeTableAddWorkTables.cs [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                    ManagerHours = table.Column<float>(nullable: false),
+                    Description = table.Column<string>(nullable: true),
+                    ModifiedAtUtc = table.Column<DateTime>(nullable: true),
```
Comment (BelehovEgor) [inline src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210805151400_DropWorkTimeTableAddWorkTables.cs:30]: Add CreatedAtUtc

Comment (BelehovEgor) [reply src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210805151400_DropWorkTimeTableAddWorkTables.cs:30]: and CreatedBy

File: src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210805151400_DropWorkTimeTableAddWorkTables.cs [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+                name: "WorkTimes");
+
+            builder.CreateTable(
+                name: DbWorkTime.TableName,
+                columns: table => new
+                {
+                    Id = table.Column<Guid>(nullable: false),
+                    UserId = table.Column<Guid>(nullable: false),
+                    ProjectId = table.Column<Guid>(nullable: false),
+                    WorkTimeMonthLimitId = table.Column<Guid>(nullable: false),
```
Comment (BelehovEgor) [inline src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210805151400_DropWorkTimeTableAddWorkTables.cs:24]: what does it mean

File: src/TimeService.Models.Db/DbWorkTimeMonthLimit.cs [hunk 5]
```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.TimeService.Models.Db
+{
+    public class DbWorkTimeMonthLimit
```
Comment (BelehovEgor) [inline src/TimeService.Models.Db/DbWorkTimeMonthLimit.cs:8]: please write me for what this entity

**Review by BelehovEgor — APPROVED** [review]

File: src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210805151400_DropWorkTimeTableAddWorkTables.cs (hunk 3 above)
Comment (MariyaSin) [reply src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210805151400_DropWorkTimeTableAddWorkTables.cs:30]: this data is generated automatically

File: src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210805151400_DropWorkTimeTableAddWorkTables.cs (hunk 4 above)
Comment (MariyaSin) [reply src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210805151400_DropWorkTimeTableAddWorkTables.cs:24]: removed

File: src/TimeService.Models.Db/DbWorkTimeMonthLimit.cs (hunk 5 above)
Comment (MariyaSin) [reply src/TimeService.Models.Db/DbWorkTimeMonthLimit.cs:8]: this is the norm hours of month

**Review by VLiamin — APPROVED** [review]

File: src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210805151400_DropWorkTimeTableAddWorkTables.cs (hunk 1 above)
Comment (MariyaSin) [reply src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210805151400_DropWorkTimeTableAddWorkTables.cs]: done

File: src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210805151400_DropWorkTimeTableAddWorkTables.cs (hunk 2 above)
Comment (MariyaSin) [reply src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210805151400_DropWorkTimeTableAddWorkTables.cs]: done

**Review by borovHokage — APPROVED** [review]

**Review by ndoriomedov — APPROVED** [review]

**Pull request merged**